### PR TITLE
Add activate scripts

### PR DESCRIPTION
**Pull request by freundTech**
This sets the typst package path to be inside the conda environment. Note that typst currently only supports one package path, so setting this will disable the default package path at
~/.local/share/typst/packages.

If you install typst using `pixi global install` you might want to pass `--no-activation` to keep the global package path.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by freundTech** [comment]
@conda-forge-admin, please rerender

File: recipe/scripts/activate-win.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export TYPST_PACKAGE_PATH_CONDA_BACKUP=${TYPST_PACKAGE_PATH:-}
+export TYPST_PACKAGE_PATH=$CONDA_PREFIX/Library/lib/jvm
```
Comment (pavelzw) [inline recipe/scripts/activate-win.sh]: Why jvm?

```suggestion
export TYPST_PACKAGE_PATH=$CONDA_PREFIX/Library/share/typst/packages
```

File: recipe/scripts/deactivate.sh
Comment (pavelzw) [inline recipe/scripts/deactivate.sh:1]: Don't you need to unset TYPST_PACKAGE_PATH here?

Comment (freundTech) [reply recipe/scripts/deactivate.sh:1]: I don't think so. If we only touch `TYPST_PACKAGE_PATH` if it was set before, in which case we also set `TYPST_PACKAGE_PATH_CONDA_BACKUP`.

Comment (freundTech) [reply recipe/scripts/deactivate.sh:1]: Also see the comment in https://github.com/conda-forge/openjdk-feedstock/blob/7161066a7f8ba00218d5c99d6a2901d37747d2ba/recipe/scripts/deactivate.sh, from where I adapted the scripts.

**Pull request merged**